src/Gui/Windows/CombinedCodeViewInteractor.cs

```diff
@@ -449,8 +451,21 @@ private void EditDeclaration()
             declarationFormInteractor.Show(screenPoint, program, addr);
         }
 
+        private Address GetCommentAnchorAddress()
+        {
+            if (combinedCodeView.MixedCodeDataView.Focused)
+                return combinedCodeView.MixedCodeDataView.GetAnchorAddress();
+            return null;
+        }
+
         private void EditComment()
         {
+            var addr = GetCommentAnchorAddress();
```

Comment:
`Edit comment` should be enabled for both the `MixedCodeControl` and the `NestedText` contrl on the right.

Reply:
Do you mean `GetCommentAnchorAddress` should look like
```
private void EditComment()
{
    if (combinedCodeView.CodeView.Focused)
        return CodeView_GetAnchorAddress();
    if (combinedCodeView.MixedCodeDataView.Focused)
        return combinedCodeView.MixedCodeDataView.GetAnchorAddress();
    return null;
}
```

Reply:
Yes. There should be a way of editing the comments on the "right side" too